general_tools/smartquotes.py

```diff
@@ -16,13 +16,14 @@
 from __future__ import unicode_literals
 import shlex
 from subprocess import *
+import sys
 
 
 def smartquotes(text):
     """
     Runs text through pandoc for smartquote correction.
     """
-    command = shlex.split('/bin/pandoc --smart -t plain')
+    command = shlex.split('pandoc --smart -t plain')
```

Comment:
@richmahn or @bspidel, is this change OK or will it break something on Lambda? I agree with the change, but since the original path is not the normal place to look for an executable file I want to be sure.